object-details/design.md

```diff
@@ -0,0 +1,29 @@
+# Object details
+
+
+## Sidecar field
+![](img/sidecar-suggestion.png)
+- We could add a `sidecar` field to the item details to illustrate whether the sidecar is missing or represent
+
+## Minor page structure improvements
+These changes should be relatively simple to make, but improve the information hierarchy of object details pages.
+
+![](img/minimal-changes.png)
+- Move page title and refresh button to above tabs
+- Remove extraneous items from breadcrumbs
+- Remove card border from object details
+
+![](img/minimal-changes-alt.png)
+- This structure would make it more appropriate to include states like "missing sidecar" in the header of the page
+
+## Moderate style changes
+Adds more space to the page and brings it in line with other cloud software
+
+![](img/moderate-changes.png)
```

Comment:
LOVE that. can you elaborate on the technical aspect? where do we get details about which font/what kind of styling to apply?

Reply:
This view is essentially using the exact same styles as OKD uses for its object detail views. I'm not sure how difficult it would be to reuse their stylesheets, but I'm sure we can put together a spec if this is the road we want to go down